website/payments/services.py

```diff
@@ -91,3 +91,10 @@ def revoke_old_mandates() -> int:
     return BankAccount.objects.filter(
         last_used__lte=(timezone.now() - timezone.timedelta(days=36 * 30))
     ).update(valid_until=timezone.now().date())
+
+
+def revoke_user_bank_account(bank_account: BankAccount) -> bool:
```

Comment:
Where is this used exactly?

Reply:
That was an oopsie by me from when I first tried to build it